app/views/features/_related.html.erb

```diff
@@ -12,7 +13,7 @@
       <section class="panel panel-content">
         <div class="panel-body">
           <p>
-          <strong><%= feature_label %></strong> has <strong class="relatedCountContainer"></strong> subordinate subjects. You can browse those subordinate subjects as well as its superordinate categories with the tree below. See the summary tab if you instead prefer to view only  its immediately subordinate subjects grouped together in useful ways, as well as   subjects non-hierarchically related to it.
+          <strong><%= feature_label %></strong> has <strong class="relatedCountContainer"></strong> subordinate <%= Feature.model_name.human(count: related_features_count).s %>. <%= ts('snippet.feature.browse_related', count: related_features_count) %> See the summary tab if you instead prefer to view only  its immediately subordinate subjects grouped together in useful ways, as well as   subjects non-hierarchically related to it.
```

Comment:
Why still have javascript populate content for `<strong class="relatedCountContainer"></strong>` instead of just putting `<strong><%= related_features_count %></strong>`?